src/app/dialog/dialog.service.ts

```diff
@@ -1,5 +1,17 @@
 // eslint-disable-next-line max-len
```

Comment:
```suggestion
/* @format */
```
Since this file is being formatted by Prettier, let's add the Prettier format pragma here.
(Also, we can get rid of that ugly eslint disable comment since... well, the line has been wrapped now and we don't want to ignore eslint!)